providers/docker/units/docker.pxu

```diff
@@ -249,10 +249,12 @@ command:
  export -n PYTHONHOME PYTHONUSERBASE PYTHONPATH
  set -e
  compose_file={root_dir}/docker-compose.yaml
- echo "test:
-   image: ubuntu
-   command: bash
-   tty: true" > $compose_file
+ echo "version: '2'
+   services:
+     test:
```

Comment:
@yphus 

the expected result from the above is yaml file that looks like this:

```yaml
version: '1'
services:
  test:
    image: ubuntu
    command: bash
    tty: true
```

Meaning version & services are top level keys. The error messages in the checkbox output suggest that above is producing yaml of this form

```yaml
version: '2'
  services:
    test:
      ....
```
as if services is not a top level key.